### DISCUSSION TITLE
Has anyone had issues updating via homebrew?

### DISCUSSION THREAD

**Discussion by pcuellar** [Eloston/ungoogled-chromium#2293] [Q&A]
I tried to update via Homebrew but it is not picking up the update for some reason. My current Chromium version is Version 110.0.5481.177. I'm on Ventura 13.3 (22E252)
<img width="746" alt="Screenshot 2023-03-29 at 8 48 53 PM" src="https://user-images.githubusercontent.com/20482146/228701226-58a8c705-02b5-40a0-9f4c-37ac0a2cc008.png">

**Comment by networkException** [comment]
The homebrew package is currently unmaintained but will point at the releases here in the future. See https://github.com/ungoogled-software/ungoogled-chromium-macos/pull/100#issuecomment-1475425335

**Comment by danni-storm** [comment]
@networkException, sorry to bother you but I am not sure the MacOS situation is very clear right now to people not super familiar with the project (i.e. myself).

After digging into the [Releases](https://github.com/ungoogled-software/ungoogled-chromium-macos/releases/tag/115.0.5790.170-1.1_arm64__1691392844) and the [Homebrew Cask](https://github.com/search?q=repo%3AHomebrew%2Fhomebrew-cask%20ungoogled&type=code) I can't find any current binaries for MacOS. I can see some of the releases in this repo targeted 116.x for MacOS but those updates do not seem to have binaries attached to them. Additionally, [this](https://github.com/ungoogled-software/ungoogled-chromium-macos/pull/119) was opened 3 weeks ago indicating MacOS builds will no longer be maintained.

Considering how old the current `eloston-chromium` build is (`115.0.5790.170-1.1,1691392844`), are there plans to remove the [current recommendation](https://github.com/ungoogled-software/ungoogled-chromium#downloads) to use the Homebrew build of ungoogled-chromium? I may also be missing something obvious, which is why I didn't open an issue here in the main repo. 

I can build for myself myself, so either way is not a huge problem for me. I just wanted a bit of clarification as running `brew upgrade` is substantially easier and fits with my daily workflow. 😅

**Reply by PF4Public** [reply]
If you can build yourself, have you considered maintaining ungoogled-chromium for Mac?

**Reply by danni-storm** [reply]
Ha, fair question. Not sure I have the time in my schedule to commit to reliably getting every build done on time. I am intrigued by the idea though.

**Reply by PF4Public** [reply]
>Not sure I have the time in my schedule to commit to reliably getting every build done on time.

This is precisely the answer to your question :)

If you could do only major releases, it would still be better compared to how it is presently.
